nomenclature/region_mapping_models.py

```diff
@@ -151,6 +153,20 @@ def create_from_region_mapping(cls, file: Union[Path, str]):
             mapping_input["common_regions"] = common_region_list
         return cls(**mapping_input)
 
+    @property
+    def region_names_all(self) -> List[str]:
```

Comment:
Given that we have an attribute `native_regions` and an attribute `common_regions`, I suggest for `all_regions` here...

Reply:
Done